src/Cake.Frosting/Extensions/CakeHostExtensions.cs

```diff
@@ -152,7 +153,7 @@ public static CakeHost UsePackageInstaller<TPackageInstaller>(this CakeHost host
         /// <param name="host">The <see cref="CakeHost"/> to configure.</param>
         /// <param name="uri">The tool URI.</param>
         /// <returns>The same <see cref="CakeHost"/> instance so that multiple calls can be chained.</returns>
-        public static CakeHost UseTool(this CakeHost host, Uri uri)
+        public static CakeHost InstallTool(this CakeHost host, Uri uri)
```

Comment:
LGTM. I'm assuming v1.0 we're OK with breaking public APIs

Reply:
Yeah, we're already breaking some APIs 😅

Reply:
We should create an additional issue for it though so that it will be listed in release notes.

Reply:
@pascalberger +1 https://github.com/cake-build/cake/issues/3050 & https://github.com/cake-build/website/issues/1504